src/components/Search/CompareOverTime.tsx

```diff
@@ -51,39 +51,26 @@ function CompareOverTime({
     !isBaseSearch && isBaseSearch !== null ? 'expanded' : 'hidden';
 
   const [timeRangeValue, setTimeRangeValue] = useState(intervalValue);
-  const [frameworkId, setframeWorkValue] = useState(frameworkIdVal);
+
   const [inProgressRevs, setInProgressRevs] = useState<Changeset[] | []>(
     newRevs,
   );
   const [baseRepository, setBaseRepository] = useState(baseRepo);
   const [newRepository, setNewRepository] = useState(newRepo);
   const [formIsDisplayed, setFormIsDisplayed] = useState(!hasEditButton);
   const mode = useAppSelector((state) => state.theme.mode);
+  const [searchParams] = useSearchParams();
   const styles = CompareCardsStyles(mode);
   const dropDownStyles = SearchStyles(mode);
   const hasCancelButton = hasEditButton && formIsDisplayed;
+  const frameworkFromURL = searchParams.get('framework');
```

Comment:
I guess you didn't see my previous message in https://github.com/mozilla/perfcompare/pull/691#discussion_r1705512934 :)

I don't understand why you're not using `frameworkIdVal` coming as a prop.

(same for `CompareWithBase`).

Reply:
Aggghhh, :/ I was doing something with the hidden input and forgot to resolve.